lnbits/core/templates/admin/_tab_exchange_providers.html

```diff
@@ -4,8 +4,8 @@ <h6 class="q-my-none q-mb-sm">
   </h6>
 
   <div class="row">
-    <div class="col-md-8 col-sm-12">
-      <div class="q-pa-sm">
+    <div class="col-12 col-md-8">
+      <div class="q-pa-sm" style="max-width: 100%; overflow: hidden">
```

Comment:
I'd lose the inline styles! it seems not needed or there's quasar classes for them.

`full-width` and `overflow-hidden`, although they don't seem needed.